docs/docs/user-guide/using-cli.md

```diff
@@ -75,10 +75,10 @@ To check if CLI detected the SPO URL previously, use the `m365 spo get` command.
 
 ## Passing complex content into CLI options
 
-When passing complex content into CLI options, such as JSON strings, you will need to properly escape nested quotes. The exact way to do it, depends on the shell that you're using. Alternatively, you can choose to pass complex content by storing the complex content in a file and passing the path to the file prefixed with an `@`, for example:
+When passing complex content into CLI options, such as JSON strings, you will need to properly escape nested quotes. The exact way to do it, depends on the shell that you're using. Alternatively, you can choose to pass complex content by storing the complex content in a file and passing the path to the file prefixed with an `` `@`` that needs to be escaped in PowerShell, for example:
```

Comment:
I'd suggest that we keep the paragraph generic and extend it with an extra sentence specific to PowerShell. Also, the sample below, I'd keep as-is and then add another sample specific to PowerShell, for which we should also denote the snippet's language as PowerShell rather than bash (`sh`) just to be explicit that `@` works in any shell and when used it PowerShell it needs to be escaped.